src/main/java/org/zone/region/ZoneManager.java

```diff
@@ -141,6 +168,7 @@
      * @param zone The zone to add
      */
     public synchronized void register(@NotNull Zone zone) {
+        this.zones.clear();
```

Comment:
Why are you clearing the zones when registering one? This would mean that there is only ever one zone?

Reply:
Uhhuh